src/extension/features/accounts/reconcile-balance/index.ts

```diff
@@ -49,37 +50,39 @@ export class ReconcileBalance extends Feature {
     }
   }
 
-  observe(changedNodes) {
+  observe(changedNodes: Set<string>) {
     if (!this.shouldInvoke()) return;
 
+    console.log(changedNodes);
+
     // When the reconciled balance icon changes, reevaluate our balance
-    if (changedNodes.has('is-reconciled-icon svg-icon lock')) {
+    if (
+      changedNodes.has('is-reconciled-icon svg-icon lock') ||
+      changedNodes.has('accounts-header-reconcile button')
+    ) {
       this.invoke();
     }
   }
 
   /**
+   *
    * Calculate the a given accounts reconciled balance
-   * @param {String} accountId The account id to get the reconciled balance for
-   * @returns {Number} balance The reconciled balance of the account
+   *
    */
-  _calculateReconciledBalance = (accountId) => {
+
+  _calculateReconciledBalance = (accountId: string) => {
     const account = getEntityManager().getAccountById(accountId);
 
     return account.getTransactions().reduce((reduced, transaction) => {
-      if (transaction.cleared && !transaction.isTombstone && transaction.isReconciled()) {
+      if (transaction.cleared && !transaction.isTombstone && transaction.isReconciled?.()) {
```

Comment:
why this change?

Reply:
Change is necessary for TS conversion. We made `isReconciled` optional incase it every disappears from the prototype.  The `?` is technically an undefined check.